index.js

```diff
@@ -169,6 +169,7 @@ module.exports.hexToUint8 = function (hex/* : string */ = '') {
 }
 
 // For browserify
+/* istanbul ignore if */
 if (typeof window === 'object') {
```

Comment:
this is exercisable if we ran these tests in a browser using `tape-run`, but probably ok for now to ignore